lib/routes/charge/items/item_page.dart

```diff
@@ -14,29 +14,53 @@ import 'package:breez/widgets/single_button_bottom_bar.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 
-class CreateItemPage extends StatefulWidget {
+import '../currency_wrapper.dart';
+
+class ItemPage extends StatefulWidget {
+  final Item item;
+  final AccountModel accountModel;
   final PosCatalogBloc _posCatalogBloc;
 
-  CreateItemPage(this._posCatalogBloc);
+  ItemPage(this._posCatalogBloc, {this.item, this.accountModel});
 
   @override
   State<StatefulWidget> createState() {
-    return CreateItemPageState();
+    return ItemPageState();
   }
 }
 
-class CreateItemPageState extends State<CreateItemPage> {
+class ItemPageState extends State<ItemPage> {
   final GlobalKey<ScaffoldState> _scaffoldKey = GlobalKey<ScaffoldState>();
 
   final _formKey = GlobalKey<FormState>();
   TextEditingController _nameController = TextEditingController();
   TextEditingController _priceController = TextEditingController();
+  TextEditingController _skuController = TextEditingController();
   AccountBloc _accountBloc;
   bool _isInit = false;
-  bool _isFiat = false;
-  Currency _selectedCurrency = Currency.BTC;
-  FiatConversion _selectedFiatCurrency;
-  String currency;
+  bool _inEditMode = false;
```

Comment:
can be calculated property.

Reply:
True, I kept this to make the code readable. I'll add a comment explaining this instead.